https://bugs.gentoo.org/show_bug.cgi?id=661880
Avoid compilatoin breakage with system-icu:
error: no matching function for call to 'ArraySizeHelper':
zero-length arrays are not permitted in C++ 
--- a/third_party/blink/renderer/platform/text/character_property_data_generator.h
+++ b/third_party/blink/renderer/platform/text/character_property_data_generator.h
@@ -244,10 +244,12 @@ static const UChar32 kIsHangulRanges[] = {
     0xD7B0, 0xD7FF,
     // Halfwidth Hangul Jamo
     // https://www.unicode.org/charts/nameslist/c_FF00.html
-    0xFFA0, 0xFFDC,
+    0xFFA0, 0xFFDB,
 };
 
-static const UChar32 kIsHangulArray[] = {};
+static const UChar32 kIsHangulArray[] = {
+  0xFFDC,
+};
 
 }  // namespace blink
 
